acceptance/indy_acceptance/indy-tests/test_scripts/acceptance_tests/keyrings_wallets.py

```diff
@@ -0,0 +1,52 @@
+"""
+Created on Nov 8, 2017
+
+@author: khoi.ngo
+
+Containing test script of test scenario 04: keyrings wallets.
+"""
+# !/usr/bin/env python3.6
+import json
+import os.path
+from indy import signus
+from indy.error import IndyError
+from libraries.constant import Constant, Colors
+from libraries.result import Status
+from libraries.common import Common
+from libraries.utils import exit_if_exception, perform
+from test_scripts.test_scenario_base import TestScenarioBase
+
+
+class TestScenario04(TestScenarioBase):
+
+    async def execute_test_steps(self):
+        print("Test Scenario 04 -> started")
+        try:
```

Comment:
These `try / except` block and logging the whole test start / finish statuses are common for all the test scenarios. They should be moved to `TestScenarioBase`. For this an attribute `name` should be introduced in `TestScenarioBase`. The value for it must be provided when initializing a concrete test scenario class.

Reply:
Please make it so

Reply:
Updated. Please refer to the latest code.